src/apps/MASS3DPA.cpp

```diff
@@ -0,0 +1,77 @@
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+// Copyright (c) 2017-20, Lawrence Livermore National Security, LLC
+// and RAJA Performance Suite project contributors.
+// See the RAJAPerf/COPYRIGHT file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+
+#include "MASS3DPA.hpp"
+
+#include "RAJA/RAJA.hpp"
+
+#include "common/DataUtils.hpp"
+
+namespace rajaperf
+{
+namespace apps
+{
+
+
+MASS3DPA::MASS3DPA(const RunParams& params)
+  : KernelBase(rajaperf::Apps_MASS3DPA, params)
+{
+  m_NE_default = 924385;
+
+  setDefaultSize(m_NE_default);
+  setDefaultReps(50);
+
+  setUsesFeature(Launch);
```

Comment:
So I picked the label for this RAJA feature based on its name in the code. Would you rather call it "teams"?

Reply:
I think we should go with teams, it's been adopted colloquially and reminds folks about computing with hierarchical parallelism. I can make that change here.